zimbra/src/main/java/fr/openent/zimbra/controllers/ZimbraController.java

```diff
@@ -94,7 +89,8 @@ public class ZimbraController extends BaseController {
     private Storage storage;
     private EventStore eventStore;
     private WorkspaceHelper workspaceHelper;
-    private CalendarServiceImpl calendarServiceImpl;
+    private QueueServiceImpl queueServiceImpl;
```

Comment:
```suggestion
    private QueueService queueService;
```
Utilise plutôt l'interface